docs/app/components/content/SupportedLanguages.vue

```diff
@@ -17,6 +17,7 @@ function getEmojiFlag(locale: string): string {
     ja: 'jp',
     ko: 'kr',
     uk: 'ua',
+    vn: 'vi',
```

Comment:
Shouldn't this be?
```suggestion
    vi: 'vn',
```

Reply:
Yeah, `vi` is the language code, `vn` is the country.